plugins/sentinelone/komand_sentinelone/actions/fetch_file_by_agent_id/action.py

```diff
@@ -0,0 +1,30 @@
+import insightconnect_plugin_runtime
+from .schema import FetchFileByAgentIdInput, FetchFileByAgentIdOutput, Input, Output, Component
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+# Custom imports below
+
+
+class FetchFileByAgentId(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="fetch_file_by_agent_id",
+            description=Component.DESCRIPTION,
+            input=FetchFileByAgentIdInput(),
+            output=FetchFileByAgentIdOutput(),
+        )
+
+    def run(self, params={}):
+        agent_id = params.get(Input.AGENT_ID, None)
```

Comment:
The `get' method returns `None' by default. No need to set it manually 😄 
```suggestion
        agent_id = params.get(Input.AGENT_ID)
```